### PR TITLE
Fix /mnm not showing the mega template's ability

### PR DESCRIPTION
**Pull request by KrisXV**

### CODE REVIEW

File: chat-plugins/othermetas.js [hunk 1]
```diff
@@ -132,6 +132,7 @@ const commands = {
 		}
 		//////////////////////////////////////////
 		let mixedTemplate = Dex.deepClone(template);
+		mixedTemplate.abilities = megaTemplate.abilities;
```
Comment (Zarel) [inline chat-plugins/othermetas.js]: This isn't a clone...

Comment (Zarel) [reply chat-plugins/othermetas.js]:
```suggestion
		mixedTemplate.abilities = Object.assign({}, megaTemplate.abilities);
```

Comment (KrisXV) [reply chat-plugins/othermetas.js]: I don't know how I didn't do that. But shouldn't this be Dex.deepClone? Or am i missing something from your refactor commit?

Comment (Zarel) [reply chat-plugins/othermetas.js]: Good point. In practice they do the same thing, but we should probably use `Dex.deepClone` for consistency.

**Pull request merged**